mayastor/src/lvm/lvm.rs

```diff
@@ -0,0 +1,285 @@
+use tokio::process::Command;
+use crate::lvm::error::Error;
+
+use rpc::mayastor::{CreatePoolRequest, Pool, PoolType, PoolState};
+
+const PVCREATE_COMMAND: &str = "pvcreate";
+const PVS_COMMAND: &str = "pvs";
+const PVREMOVE_COMMAND: &str = "pvremove";
+const VGCREATE_COMMAND: &str = "vgcreate";
+const VGS_COMMAND: &str = "vgs";
+const VGREMOVE_COMMAND: &str = "vgremove";
+
+pub async fn lookup_lvm_vg(name: &str) -> Option<Pool> {
```

Comment:
Functions should have /// style comments to explain what they do. It gets added to our docs that way. It is clear what the function does in this case, so a simple short brief is more than sufficient.

Reply:
Also:

*  it's best to avoid using abbreviations as much possible IMO (unless they're well known even to people unfamiliar with the tech in question). Similarly for all for the identifiers, e.g `VolGroupList` (even though they're internal, it add to code readability).

* since this function is already in the `lvm` module, you don't need to have `_lvm_` in the names, right?

```suggestion
pub async fn lookup_vol_group(name: &str) -> Option<Pool> {
```